keyboards/function96/v1/mcuconf.h

```diff
@@ -0,0 +1,168 @@
+/*
```

Comment:
Please run `qmk chibios-confmigrate` on this file.

Reply:
Sounds good: What should I input as the -r (Reference) and -i (Input)?